data/GMSim.py

```diff
@@ -0,0 +1,441 @@
+from re import T
+import numpy as np
+from scipy.stats import gamma
+import sys
+# import csv
+np.set_printoptions(threshold=sys.maxsize)
+
+
```

Comment:
Can you please add comments to each function and other important stuff in this Python file? Would help future users.

Reply:
Oh I guess you already did this, right?

Reply:
Yes, the GMSim.py script is pretty well-documented